tests/server-tests/api_test.go

```diff
@@ -203,16 +257,16 @@ func TestBasicFileOps(t *testing.T) {
 		// handle error
 	}
 
+	// Clear the headers
+	testInput.headers = nil
+
 	testInput.method = "GET"
 	testInput.expectedStatusCode = 200
 	err = testMethod(t, testInput)
 	if err != nil {
 		// handle error
```

Comment:
I'm not sure if ignoring an error here is going to cause pain in debugging later.